tests/integration/security/ecc_signature_algorithm/main_test.go

```diff
@@ -1,4 +1,3 @@
-// +build integ
```

Comment:
all tests in this folder need this tag

Reply:
You mean in the integration folder?

Reply:
Yes, this tag is intended. So if you run `go test ./...` it doesn't touch the k8s cluster

Reply:
Ahh okay, this was not there in other tests so, I removed it from here too. I would add the tag to all the tests in the folder.

Reply:
I guess I would the tag to the Makefile too, since it is not there, https://github.com/istio/istio/blob/release-1.7/tests/integration/tests.mk#L92